recipes/xilinx-runtime/conda_build_config.yaml

```diff
@@ -0,0 +1,10 @@
+c_compiler:
```

Comment:
Why is this file needed? Can you add a comment or omit it?

Reply:
Our C++ code has only been verified on gcc9. By default gcc will default to gcc12 in conda-forge. So the purpose of this file is to constrain the compiler version on linux. In the future I imagine it would be used to constrain the compiler version for Windows as well. Is this not the correct approach?

Reply:
We try to build all packages with the same compiler on conda-forge. Sticking to older compilers increases maintenance (not only for the package maintainer) and should only be done if things don't work. In this case, I would suspect that compiling with the default compilers should work out of the box.